src/MSBuild/OutOfProcTaskHostNode.cs

```diff
@@ -492,6 +492,22 @@ public void ReleaseCores(int coresToRelease)
         }
 
         #endregion
+
+        #region IBuildEngine10 Members
+
+        [Serializable]
+        private sealed class BuildEngineInterfaceImpl : BuildEngineInterface
+        {
+            /// <summary>
+            /// No logging verbosity optimization in OOP nodes.
```

Comment:
```suggestion
            /// No logging verbosity optimization in TaskHost nodes.
```

? We should still apply it to worker nodes in the distributed-logger scenario, right?

Reply:
That's right, fixed, thank you!